### PR TITLE
Register devmatei.is-a.dev

### PR DESCRIPTION
**Pull request by DevMatei**
Register devmatei.is-a.dev with TXT record pointing to vercel.devmatei.is-a.dev.

### CODE REVIEW

**Comment by DevMatei** [comment]
Actualy its _vercel.devmatei.is-a.dev I did a mistake

File: domains/devmatei.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "DevMatei",
+           "email": "",
+           "discord": "784443338627612673",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Xfkro-38gTXoPdcrF-mGP-0gpbfMqsWBqeYOyYVKKsmt3ImwKpLmeMfsMeYxXLzyjVCRhnSUPGYYvoqg773HsJGh8dLnyuCI7SgiolIqnb2pS7t3xyxNX2sq0QBPPqlGLzDGhRYMVjIq02aNIksdh3XiJ3vsS36yVxJ4bOb4gG-t4y6hqmxHLICK7wPwG3zVQcqse63QtEyNdPpmDNbzmcmp3pxDzD-cmrX8FphYTcABbL7pphEZyZNMTtVaR-mfKafVmd2pmE_L-J3EKNSzLJj-8HSFx5N4KN36uWs01sBKHWRWz5Co6YRvT3eQxgu9WYuocLlFrZ8aPqQcsCkXGg.YVkATTXNM6fkDeg4Ew01JA.YXGkCx-tdRkCp2th5RJLnEsOUSmisdUKE4KbpcxG9ZVVodzbkbDXwLwjqTntT_5xqe84ixhZfRCb-TAGAYkWyopaG3EpnWtWYkLNB43Vpfo.8xLIYHMxurOHww7iUeAqMQ"
+        },
+    
+        "record": {
+            "TXT": "vercel.devmatei.is-a.dev"
```
Comment (Stef-00012) [inline domains/devmatei.json:10]: TXT records alone do nothing

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/devmatei.json (hunk 1 above)
Comment (DevMatei) [reply domains/devmatei.json:10]: Actualy its _vercel.devmatei.is-a.dev I did a mistake

**Comment by CuteDog5695** [comment]
Did you mean to put your PR on draft?

**Comment by DevMatei** [comment]
> Did you mean to put your PR on draft?

nah mb

File: domains/devmatei.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "DevMatei",
+           "email": "",
+           "discord": "784443338627612673",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Xfkro-38gTXoPdcrF-mGP-0gpbfMqsWBqeYOyYVKKsmt3ImwKpLmeMfsMeYxXLzyjVCRhnSUPGYYvoqg773HsJGh8dLnyuCI7SgiolIqnb2pS7t3xyxNX2sq0QBPPqlGLzDGhRYMVjIq02aNIksdh3XiJ3vsS36yVxJ4bOb4gG-t4y6hqmxHLICK7wPwG3zVQcqse63QtEyNdPpmDNbzmcmp3pxDzD-cmrX8FphYTcABbL7pphEZyZNMTtVaR-mfKafVmd2pmE_L-J3EKNSzLJj-8HSFx5N4KN36uWs01sBKHWRWz5Co6YRvT3eQxgu9WYuocLlFrZ8aPqQcsCkXGg.YVkATTXNM6fkDeg4Ew01JA.YXGkCx-tdRkCp2th5RJLnEsOUSmisdUKE4KbpcxG9ZVVodzbkbDXwLwjqTntT_5xqe84ixhZfRCb-TAGAYkWyopaG3EpnWtWYkLNB43Vpfo.8xLIYHMxurOHww7iUeAqMQ"
+        },
+    
+        "record": {
+            "TXT": "_vercel.devmatei.is-a.dev"
```
Comment (CuteDog5695) [inline domains/devmatei.json:10]: TXT does nothing, if you want to use Vercel then please check the docs: https://www.is-a.dev/docs/vercel/

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by DevMatei** [comment]
> stale

💔

**Pull request closed without merging**